### PR TITLE
[REVIEW] bugs: 1939, 1909, 1890, 1887, 1886, 1487, 1426 and 1112 tests associated

### PR DESCRIPTION
**Pull request by iariasleon**
@fgalan @kzangeli @crbrox

### CODE REVIEW

File: test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature [hunk 1]
```diff
@@ -753,7 +752,7 @@ Feature: get an attribute value by entity ID and attribute name if it exists usi
       | error       | BadRequest        |
       | description | service not found |
 
-  @entity_id_wrong
+  @entity_id_wrong  @BUG_1887 @skip
```
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature]: This is still `@skip`?

File: test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature [hunk 2]
```diff
@@ -1031,23 +1032,33 @@ Feature: get an attribute value by entity ID and attribute name if it exists usi
...
+      | Fiware-ServicePath | /test                 |
+      | Accept             | text/plain            |
+    When get an attribute value by ID "room_1" and attribute name "house_/" if it exists
+    Then verify that receive an "Bad Request" http code
+    And verify an error response
+      | parameter   | value             |
+      | error       | BadRequest        |
+      | description | service not found |
+
   @attribute_name_invalid @BUG_1887 @skip
```
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature]: This is still `@skip`?

Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature]: yes, it is scenario and another with invalid id using `house_#` return an 406 http code. Commented in the issue #1887

File: test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature]: yes, it is scenario and another with invalid attr_name using house_# return an 406 http code. Commented in the issue #1887

File: test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature (hunk 2 above)
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature]: NTC

File: test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature (hunk 1 above)
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature]: NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**